docs/wiki/99 - Developers/Automated Testing.md

```diff
@@ -91,7 +91,7 @@ We use Docker to help us simulate a server running LORIS. The installation and u
 
 LORIS uses the [PHPUnit library](https://phpunit.de/) for unit tests.
 
-Unit testing is covered in depth in our [Unit Test Guide](test/UnitTestGuide.md).
+Unit testing is covered in depth in our [Unit Test Guide](../../../test/UnitTestGuide.md).
```

Comment:
I think these changes were already applied in https://github.com/aces/Loris/pull/6192

You may need to rebase your pull request so that the changes are properly applied. If you have any questions about this let me know and I'll provide some instructions.